groups/utils.js

```diff
@@ -104,6 +104,24 @@ async function addGroupRoleToMember(memberRoleBody) {
   }
 }
 
+async function removeRoleFromMember(roleId, discordId) {
+  try {
+    const res = await fetch(`${BASE_URL}/discord-actions/roles`, {
+      method: 'DELETE',
+      credentials: 'include',
+      headers: {
+        'Content-type': 'application/json',
+      },
+      body: JSON.stringify({ roleid: roleId, userid: discordId }),
+    });
+
+    return await res.json();
+  } catch (err) {
+    console.error(err);
```

Comment:
why did you use console here

Reply:
logger not implemented on dashboard site